"""fix category error 

Revision ID: 10faf3814a61
Revises: 4f58873a9592
Create Date: 2025-05-10 14:48:22.405482

"""
from typing import Sequence, Union

from alembic import op
import sqlalchemy as sa


# revision identifiers, used by Alembic.
revision: str = '10faf3814a61'
down_revision: Union[str, None] = '4f58873a9592'
branch_labels: Union[str, Sequence[str], None] = None
depends_on: Union[str, Sequence[str], None] = None


def upgrade() -> None:
    """Upgrade schema."""
    # ### commands auto generated by Alembic - please adjust! ###
    op.add_column('startup', sa.Column('category_id', sa.Integer(), nullable=True))
    op.create_foreign_key(None, 'startup', 'category', ['category_id'], ['id'], ondelete='cascade')
    # ### end Alembic commands ###


def downgrade() -> None:
    """Downgrade schema."""
    # ### commands auto generated by Alembic - please adjust! ###
    op.drop_constraint(None, 'startup', type_='foreignkey')
    op.drop_column('startup', 'category_id')
    # ### end Alembic commands ###
